pkg/channel/message_receiver.go

```diff
@@ -72,7 +72,6 @@ type MessageReceiverOptions func(*MessageReceiver) error
 
 // ResolveChannelFromHostFunc function enables EventReceiver to get the Channel Reference from incoming request HostHeader
 // before calling receiverFunc.
-// Returns UnknownHostError if the channel is not found, otherwise returns a generic error.
 type ResolveChannelFromHostFunc func(string) (ChannelReference, error)
```

Comment:
This type is no longer accurate as it's not `ResolveChannelFromHost` but rather `ResolveChannelFromURL`. I left it as is since other implementors (like natss) rely on it.